packages/x-components/src/x-modules/search-box/components/search-input-placeholder.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <label ref="placeholder" class="placeholder"></label>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue';
+  import { Component, Prop } from 'vue-property-decorator';
+  import { State } from '../../../components/decorators/store.decorators';
+  import { XOn } from '../../../components/decorators/bus.decorators';
+  import { xComponentMixin } from '../../../components/x-component.mixin';
+  import { searchBoxXModule } from '../x-module';
+  import SearchInput from './search-input.vue';
+  @Component({
+    components: { SearchInput },
+    mixins: [xComponentMixin(searchBoxXModule)]
+  })
+  export default class SearchInputPlaceholder extends Vue {
+    @State('searchBox', 'query')
+    public query!: string;
+
+    @Prop({ default: () => [''] })
+    protected placeholdersArray!: string[];
+
+    protected get isQueryEmpty(): boolean {
+      return this.query.length === 0;
+    }
+
+    @XOn(['UserHoveringOverSearchBox'])
```

Comment:
I think there's no need to create new events to handle this. You can just listen to the mouse events on this component.